src/plugins/intel_cpu/tests/functional/single_layer_tests/matmul.cpp

```diff
@@ -824,6 +824,54 @@ const auto testParams2D_Brgemm_Amx_smoke = ::testing::Combine(fullyConnectedPara
 
 INSTANTIATE_TEST_SUITE_P(smoke_FC_2D_Brgemm_Amx, MatMulLayerCPUTest, testParams2D_Brgemm_Amx_smoke, MatMulLayerCPUTest::getTestCaseName);
 
+#ifdef OV_CPU_WITH_LLMDNN
```

Comment:
Do you try all the tests on SPR machine ? Does `llmdnn` affects normal brgemm fusing tests ?

Reply:
No, no new failed case found.